### ISSUE TITLE
trying to deploy your example

### ISSUE THREAD

**Issue by mathieujobin** [leafo/heroku-lapis-example#1]
Hello,

is this example suppose to work out of the box?

I just tried a simple clone and to push it to a new application container on heroku but it fails to detect the app

```
[mathieu@hz550 heroku-lapis-example]$ git push heroku 
Counting objects: 32, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (16/16), done.
Writing objects: 100% (32/32), 4.77 KiB | 0 bytes/s, done.
Total 32 (delta 12), reused 32 (delta 12)
remote: Compressing source files... done.
remote: Building source:
remote: 
remote: 
remote:  !     Push rejected, no Cedar-supported app detected
remote: HINT: This occurs when Heroku cannot detect the buildpack
remote:       to use for this application automatically.
remote: See https://devcenter.heroku.com/articles/buildpacks
remote: 
remote: Verifying deploy...
remote: 
remote: !       Push rejected to heroku-lapis-example.
remote: 
To https://git.heroku.com/heroku-lapis-example.git
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'https://git.heroku.com/heroku-lapis-example.git'
```

a little bit more documentation would be appreciated

thanks for pointing me out in the right direction.

**Comment by mathieujobin** [comment]
I trashed this first heroku app, found more documentation in the buildpack project.
so i created a new heroku app 

``` bash
heroku create --buildpack http://github.com/leafo/heroku-buildpack-lua.git
```

and I got further, I started updating the README with more info
but the example is not working out of the box.

lapis cannot be found.

I tried to run `luarocks install lapis` first, but luarocks could not be found.

I tried `lua web.lua $PORT` but dependencies still could not be found...

I tried updating the openresty rockspec to `heroku-openresty-dev-2.rockspec` as per https://github.com/leafo/heroku-openresty/pull/10 but no luck
